packages/react/src/class-names/index.ts

```diff
@@ -3,11 +3,18 @@ import type { ReactNode, CSSProperties } from 'react';
 import type { BasicTemplateInterpolations, CssFunction } from '../types';
 import { createSetupError } from '../utils/error';
 
-export type Interpolations = (BasicTemplateInterpolations | CssFunction | CssFunction[])[];
+export type Interpolations = (
+  | BasicTemplateInterpolations
+  | CssFunction<unknown>
```

Comment:
nit: Is it possible for `Interpolations` to have generics so we don't have to use `unknown` here?

Reply:
Yep, will do